src/handler.rs

```diff
@@ -148,7 +155,32 @@ pub enum Command {
 
 #[derive(Debug, Clone, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct SubstrateCommand {}
+pub enum SubstrateCommand {
+    MixerRelayTx(MixerRelayTransaction),
+}
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MixerRelayTransaction {
+    /// one of the supported chains of this relayer
+    pub chain: String,
+    /// The tree id of the mixer's underlying tree
+    pub id: u32,
+    /// The zero-knowledge proof bytes
+    pub proof: Vec<u8>,
+    /// The target merkle root for the proof
+    pub root: Element,
+    /// The nullifier_hash for the proof
+    pub nullifier_hash: Element,
+    /// The receipient of the transaction
+    pub recipient: AccountData,
```

Comment:
This one is a bit tricky here, the way we send it to the verifier on the withdraw is that we use custom encoders to make the `AccountId` into a 32 bytes `Element`. so again you should make it `H256` so the client has to format it, or better use `subxt::sp_core::crypto::AccountId32`
```suggestion
    pub recipient: subxt::sp_core::crypto::AccountId32,
```